test/pylib/scylla_cluster.py

```diff
@@ -601,18 +605,21 @@ def report_error(message: str):
             if self.cmd.returncode:
                 self.cmd = None
                 if expected_error is not None:
-                    with self.log_filename.open('r') as log_file:
+                    with self.log_filename.open("r", encoding="utf-8") as log_file:
                         for line in log_file:
                             if expected_error in line:
                                 return
                         report_error("the node startup failed, but the log file doesn't contain the expected error")
                 report_error("failed to start the node")
 
             if hasattr(self, "host_id") or await self.get_host_id(api):
```

Comment:
After this line we should assign `server_up_state = HOST_ID_QUERIED`
and then (see comments above)  `server_up_state = await self.cql_is_up(..., start_state=server_up_state)`